src/startup.rs

```diff
@@ -21,6 +21,7 @@ use crate::{
 pub struct AppState {
     pub pool: PgPool,
     pub email_client: EmailClient,
+    pub application_base_url: String,
```

Comment:
Add new setting to hold the APIs base url for when we send the confirmation email link to new subscribers. We need it to be configurable because it will differ depending on the environment